New Agent troubleshooting section (#4606)
DataDog/documentation: content/en/agent/troubleshooting/debug_mode.md

---
title: Debug Mode
kind: documentation
disable_toc: true
aliases:
    - /agent/faq/how-to-get-more-logging-from-the-agent
    - /agent/faq/agent-5-container-more-log
further_reading:
- link: "/agent/troubleshooting/send_a_flare"
  tag: "Agent Troubleshooting"
  text: "Send an Agent Flare"
- link: "/agent/troubleshooting/agent_check_status"
  tag: "Agent Troubleshooting"
  text: "Get the Status of an Agent Check"
---

To enable the Agent full debug mode:

1. Modify your local `datadog.yaml` file (see [this page][1] to locate this configuration file on your instance)

2. Replace `# log_level: INFO` with `log_level: DEBUG` (remove `#` to uncomment the line).

3. Restart the Datadog Agent. See the [Agent Commands][2] page for OS-specific details.

> 4. Wait a few minutes to generate some logs. [See the Agent][3] docuementation for the location of the logs.

Reviewer: ```suggestion
4. Wait a few minutes to generate some logs. [See the Agent][3] documentation for the location of the logs.
```

## Obtaining debug logs from the container Agent

{{< tabs >}}
{{% tab "Agent v6" %}}

**Set the `DD_LOG_LEVEL=debug` environment variable when starting your Agent.**

If your container is already running:

> 1. To avoid that your process is restarted by S6, run:

Reviewer: ```suggestion
1. To prevent your process from being restarted by S6, run:
```
